api/urls.py

```diff
@@ -0,0 +1,21 @@
+from django.urls import include, path
+from rest_framework.routers import DefaultRouter
+
+from api.views import (
+    AgeRangeViewSet,
+    PersonalInfoViewSet,
+    StudentViewSet,
+    TeacherUnder18ViewSet,
+    TeacherViewSet,
+)
+
+router = DefaultRouter()
+router.register(r"personal_info", PersonalInfoViewSet, basename="personal_info")
+router.register(r"students", StudentViewSet, basename="students")
```

Comment:
A purely aesthetic question: it's strange why "Language(s) the students and teachers can speak in class" comes before all other fields when I browse to `/api/personal_info/` or why "Comment" comes right after "Personal info" in `/api/students/`. This doesn't matter of course, I'm just curious as to how the fields are ordered.

Reply:
It's order which you defined in models' fields :D 

And if you use some nested serializers, I think, they are parsed first (you could check it after latest PR update with readable nested structures instead ids), and then model's fields are parsed

Reply:
Oh, I see, `communication_language_mode` is defined in the abstract model that people and `Group` subclass from. That's why it comes first.